stable/vmware-exporter/values.yaml

```diff
@@ -0,0 +1,75 @@
+---
+
+replicaCount: 1
+
+vsphere:
+  user: user
+  password: na
+  host: vcenter
+  ignoressl: true
+  collectors:
+    hosts: true
+    datastores: true
+    vms: true
+
+podAnnotations:
+  prometheus.io/scrape: "true"
+  prometheus.io/port: "9272"
+  prometheus.io/path: "/metrics"
+
+readinessProbe:
+  httpGet:
+    path: /healthz
+    port: 9272
+
+livenessProbe:
+  httpGet:
+    path: /healthz
+    port: 9272
+  initialDelaySeconds: 30
+  failureThreshold: 10
+
+image:
+  repository: pryorda/vmware_exporter
+  tag: v0.7.4
+  pullPolicy: IfNotPresent
+
+service:
```

Comment:
Do we need to enable service ? The scrapping from prometheus is done at the pod level.

Reply:
Being able to retrieve metrics via ingress would be great for testing and debugging purposes. I can switch this off in the defaults. Any thougts?

Reply:
Defaults (as insecure as they are) tend to stay. I'd disable

Reply:
Fixed with service.enabled: false as default